models/src_twitter_ads.yml

```diff
@@ -182,4 +188,92 @@ sources:
+        description: Each record represents the performance of an ad campaign on a given day.
+        identifier: "{{ var('twitter_ads_campaign_report_identifier', 'campaign_report') }}"
+        columns:
+          - name: _fivetran_synced
+            description: "{{ doc('_fivetran_synced') }}"
+          - name: account_id
+            description: The ID of the related account.
+          - name: billed_charge_local_micro
+            description: The spend for the campaign on that day, in micros and in whichever currency was selected during account creation.
+          - name: clicks
+            description: The clicks for the campaign on that day.
+          - name: date
+            description: The date of the performance.
+          - name: impressions
+            description: The impressions for the campaign on that day.
+          - name: campaign_id
+            description: The ID of the ad campaign.
+          - name: url_clicks
+            description: The url clicks for the campaign on that day.
+          - name: placement
+            description: Where on Twitter the ad is being displayed. Possible values include 'ALL_ON_TWITTER', 'PUBLISHER_NETWORK', 'TWITTER_PROFILE', 'TWITTER_SEARCH', 'TWITTER_TIMELINE', and 'TAP_*', which are more granular options for `PUBLISHER_NETWORK`.
+      - name: line_item_report
+        description: Each record represents the performance of a line item (ad group) on a given day.
+        identifier: "{{ var('twitter_ads_line_item_report_identifier', 'line_item_report') }}"
+        columns:
+          - name: _fivetran_synced
+            description: "{{ doc('_fivetran_synced') }}"
+          - name: account_id
+            description: The ID of the related account.
+          - name: billed_charge_local_micro
+            description: The spend for the line item on that day, in micros and in whichever currency was selected during account creation.
+          - name: clicks
+            description: The clicks for the line item on that day.
+          - name: date
+            description: The date of the performance.
+          - name: impressions
+            description: The impressions for the line item on that day.
+          - name: line_item_id
+            description: The ID of the line item.
+          - name: url_clicks
+            description: The url clicks for the line item on that day.
+          - name: placement
+            description: Where on Twitter the ad is being displayed. Possible values include 'ALL_ON_TWITTER', 'PUBLISHER_NETWORK', 'TWITTER_PROFILE', 'TWITTER_SEARCH', 'TWITTER_TIMELINE', and 'TAP_*', which are more granular options for `PUBLISHER_NETWORK`.
+      - name: line_item_keywords_report
+        description: Each record represents the performance of a line item (ad group) and keyword combination on a given day.
+        identifier: "{{ var('twitter_ads_line_item_keywords_report_identifier', 'line_item_keywords_report') }}"
+        config:
```

Comment:
Why do we use `config` here, but in other packages we use `meta`?
https://github.com/fivetran/dbt_zendesk_source/blob/aef4f3e6d436b4d5f52ecac81c8beb71951289b0/models/src_zendesk.yml#L190-L193

Reply:
presuming we're good to use config here now that we've tested it out?

Reply:
Yup all good!